code/modules/antagonists/clock_cult/scriptures/clockwork_armaments.dm

```diff
@@ -24,6 +24,9 @@
 	var/static/datum/outfit/clockcult/armaments/sword/armaments_sword = new
 	var/static/datum/outfit/clockcult/armaments/bow/armaments_bow = new
 	var/static/datum/outfit/clockcult/default = new
+	var/static/datum/outfit/clockcult/plasmaman/plasmaman = new
+	if(is_species(M, /datum/species/plasmaman))
+		plasmaman.equip(M)
```

Comment:
This will cause them to be equipped with the standard loadout gear which will override the brass gloves and shoes.